### ISSUE TITLE
Unknown error causing random crashes

### ISSUE THREAD

**Issue by Luf2K14** [AppliedEnergistics/Applied-Energistics-2#3987]
while im playing sometimes the game it self it crashes i already requested help in a Discord group 'All the Mods' and the ALM team said that is something with AE2 causing crashes

**Describe the bug**
unknown as said above

**To Reproduce**
random unfortunately... 

**Expected behavior**
to give an error in chat in-game but it crashes forcing me to end session in order to force close java sometimes (task manager is useless in that case)

**Additional context**
here is the most recent log:
https://pastebin.com/PGHNmCnK

**Environment**

- Minecraft Version: 1.12.2
- AE2 Version: appliedenergistics2-rv6-stable-6
- Forge Version: 1.12.2-forge1.12.2-14.23.5.2823

**Comment by yueh** [comment]
I currently cannot see why it would crash here. Currently I'd suspect that some addon/integration does something wrong, but no real idea. For example the ender io cables were always a bit broken.

**Comment by Luf2K14** [comment]
oh so the main issue could be Ender io not AE2 problem ... but if you want the savegame to check it i will gladly provide it :)

**Comment by yueh** [comment]
Does not have to be EnderIO. Could be pretty much any AE2 addon like AE2Stuff, etc.

**Comment by auddie2015** [comment]
I think this might be the same thing.

"a fatal error has occurred, this connection is terminated"

It happened when I placed a crafting terminal onto an ME cable. 

Forge: 14.23.5.2835 (Installed via Twitch profile)
AE2: rv6-stable-6
AE2 Stuff: 0.7.0.4
Extra Cells 2: 2.6.2
Ender IO: 5.0.43
Inventory Tweaks: 1.63+release.109.220
RFTools: 1.12-7.6.1
Mekanism: 1.12.2-9.7.0.369

This crash started this morning so I am in a test world as I have been trying to track it down. I have completely reinstalled Minecraft after removing all traces from the system, reinstalled all mods to make sure nothing is outdated, and reset the Minecraft Plugin in Twitch. 

All that is deployed is an RFTools Modular Storage, RFTools Builder, Mekanism Energy Cube, AE2 Energy Acceptor (channels are turned off so no controller), ME Drive, with two Extra Cells storage disks, a single ME cable, a crafting terminal on that cable, an ME Interface, and Ender IO item conduit connected between the storage and the interface. It is a compact recreation of the quarry setup that is crashing the internal server so I can turn things off until I isolate the problem (if I can with no crash report).

If I find anything, I will add to it.

**Comment by auddie2015** [comment]
I found my problem. Maybe it is the same for the OP since it is in many modpacks. 

The One Probe, when you pass your mouse over anything AE2, causes a fatal error in the internal server and you get disconnected. I currently have 1.12.2.1.2 of The One Probe installed.

**Comment by Dimentive** [comment]
I had a similar crash (and a “Batching chunks” crash): #4004

However, I do not have The One Probe installed, nor did I look at any of my Applied Energistics 2 blocks. I simply flew away from my base into the wilderness. A minute before I crashed, I used the wireless crafting terminal from [Wireless Crafting Terminal](https://minecraft.curseforge.com/projects/wireless-crafting-terminal), but I’m not sure if that has anything to do with it.
